integrations/oap/kubernetes/oap/sql-ds-cache/plasma-store-server.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: plasma-store-server
+  namespace: default
+  labels:
+    k8s-app: oap-cache-backend
+spec:
+  selector:
+    matchLabels:
+      name: pmem
+  template:
+    metadata:
+      labels:
+        name: pmem
+    spec:
+      containers:
+      - name: plasma-store-server
+        image: oap-centos:1.2.0
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/sh"]
+        args: ["-c", "export LD_LIBRARY_PATH=/opt/home/conda/envs/oap-1.2.0/lib:$LD_LIBRARY_PATH; echo $LD_LIBRARY_PATH; export PMEM_PATH=/mnt/pmem; export PMEM_SIZE=5000000000; /opt/home/conda/envs/oap-1.2.0/bin/plasma-store-server -m $PMEM_SIZE -d $PMEM_PATH -s /var/log/plasmaStore;"]
+        resources:
```

Comment:
Also can export /opt/home/conda/envs/oap-1.2.0 to env;
Make PMEM_SIZE passed by command